tests/pkg-layering/main.yml

```diff
@@ -307,6 +307,93 @@
       vars:
         package: "{{ g_epel_pkg }}"
 
+- name: Package Layering - GPG Test
+  hosts: all
+  become: true
+
+  tags:
+    - gpg
+
+  tasks:
+    - name: Make /srv/localrepo directory
+      file:
+        path: /srv/localrepo
+        state: directory
+        mode: 0755
+
+    - name: Copy aht-dummy rpm to /srv/localrepo
+      copy:
+         src: ../../rpm/aht-dummy-1.0-1.noarch.rpm
```

Comment:
Could you use `playbook_dir` here instead of the relative path?

http://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#magic-variables-and-how-to-access-information-about-other-hosts